web/src/components/layout/nav.css

```diff
@@ -7,7 +7,6 @@ header {
     align-items: center;
     height: var(--header-height);
     box-sizing: border-box;
-    z-index: var(--top-z-index);
```

Comment:
https://github.com/common-voice/common-voice/pull/3447/files#diff-66d4b249909eebad33f1f36e23420d6ba595ffb2534a44ed51693aca773d189dR14 Is defined couple lines down